Combobox/ComboPopup.html

```diff
@@ -1,8 +1,9 @@
 <template role="presentation"
 	requires="deliteful/LinearLayout, deliteful/Button">
 	<d-linear-layout style="height: 100%">
-        <div class="d-combo-popup-header">{{header}}</div>
+        <div class="d-combo-popup-header" id="{{_tag}}-{{widgetId}}-header">{{header}}</div>
 		<input d-hidden="{{this.combobox._inputReadOnly}}"
+			aria-labelledby="{{_tag}}-{{widgetId}}-header"
```

Comment:
Note: This confused me at first since tests/functional/ComboPopup.js already checks for a label for the input:

```
var checkListInPopup = function (remote, comboId, hasFilterInput, isMultiSelect) {
...
		.findByCssSelector("label[for='" + comboId + "-input']")
		.getVisibleText()
		.then(function (value) {
			assert.strictEqual(value, label);
			label = null;
		})
```

It turns out that test is useless though because it's checking the `<label>` for the `<input>` in the original Combobox rather than for  the `<input>` in the ComboPopup.

So, I see why you added this.  It's unusual to use aria-labelledby rather than `<label>` though.